wpilibNewCommands/src/main/native/include/frc2/command/CommandScheduler.h

```diff
@@ -164,6 +164,13 @@ class CommandScheduler final : public nt::NTSendable,
   void UnregisterSubsystem(std::initializer_list<Subsystem*> subsystems);
   void UnregisterSubsystem(std::span<Subsystem* const> subsystems);
 
+  /**
+   * Un-registers all registered Subsystems with the scheduler. All currently
+   * registered subsystems will no longer have their periodic block called, and
+   * will not have their default command scheduled.
+   */
+  void UnregisterAllSubsystems();
```

Comment:
CI seems to be failing since this is missing a definition.

Reply:
oops.